faq_bot/bot/model.py

```diff
@@ -0,0 +1,25 @@
+import json
+from pathlib import Path
+
+import numpy
+from sentence_transformers import SentenceTransformer
+
+model = SentenceTransformer('clips/mfaq')
+
+with open(Path(__file__).parent / "faq.json", "r", encoding="utf-8") as file:
+    faq = json.load(file)
+
+
+def find_similar_questions(question: str):
```

Comment:
A little bit docs string for all functions

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/7b9d6521717f1d9073a1245b8e163d1b67c41618